addons/point_of_sale/static/src/app/navbar/sale_details_button/sales_detail_report.xml

```diff
@@ -52,7 +52,7 @@
                         <div class="responsive-price">
                             <t t-esc="line['product_name'].substr(0,20)" />
                             <span class="pos-receipt-right-align">
-                                <t t-esc="Math.round(line.qty * Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure')?.digits)) / Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure')?.digits)" />
+                                <t t-esc="Math.round(line.quantity * Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure')?.digits)) / Math.pow(10, pos.models['decimal.precision'].find((dp) => dp.name === 'Product Unit of Measure')?.digits)" />
```

Comment:
this seems like it should be in a different commit

Reply:
yes it is

Reply:
:facepalm: my bad, sorry